### PR TITLE
add PP-YOLO tiny

### PR DESCRIPTION
**Pull request by heavengate**
**add PP-YOLO tiny**

### PP-YOLO tiny

|            Model             | GPU number | images/GPU | Model Size | input shape | Box AP<sup>val</sup> | Kirin 990 4xCore(FPS) | download | config  |
|:----------------------------:|:-------:|:-------------:|:----------:| :----------:| :------------------: | :-------------------: | :------: | :------: |
| PP-YOLO tiny                 |    8    |      32       |   4.2MB    |     320     |         20.6         |          92.3         | [model](https://paddledet.bj.bcebos.com/models/ppyolo_tiny_650e_coco.pdparams) | [config](https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/ppyolo/ppyolo_tiny_650e_coco.yml)                   |
| PP-YOLO tiny                 |    8    |      32       |   4.2MB    |     416     |         22.7         |          65.4         | [model](https://paddledet.bj.bcebos.com/models/ppyolo_tiny_650e_coco.pdparams) | [config](https://github.com/PaddlePaddle/PaddleDetection/tree/develop/configs/ppyolo/ppyolo_tiny_650e_coco.yml)                   |

**Notes:**

- PP-YOLO-tiny is trained on COCO train2017 datast and evaluated on val2017 dataset，Box AP<sup>val</sup> is evaluation results of `mAP(IoU=0.5:0.95)`, Box AP<sup>val</sup> is evaluation results of `mAP(IoU=0.5)`.
- PP-YOLO-tiny used 8 GPUs for training and mini-batch size as 32 on each GPU, if GPU number and mini-batch size is changed, learning rate and iteration times should be adjusted according [FAQ](https://github.com/PaddlePaddle/PaddleDetection/blob/master/docs/FAQ.md).
- PP-YOLO_MobileNetV3 inference speed is tested on Kirin 990 with 4 threads by arm8
- we alse provide PP-YOLO-tiny post quant inference model, which can compress model to 1.3MB with nearly no inference on inference speed and performance

### CODE REVIEW

**Review by qingqing01 — DISMISSED** [review]
need to add quantized 1.3M model

**Comment by heavengate** [comment]
> need to add quantized 1.3M model

Done, thanks!

**Review by qingqing01 — DISMISSED** [review]

**Review by qingqing01 — APPROVED** [review]
